lib/Backend/GlobOptIntBounds.cpp

```diff
@@ -1278,14 +1278,21 @@ GlobOpt::InvalidateInductionVariables(IR::Instr * instr)
     }
 
     // If this is an induction variable, then treat it the way the prepass would have if it had seen
-    // the assignment and the resulting change to the value number, and mark it as indeterminate.
+	// the assignment and the resulting change to the value number, and mark induction variables
```

Comment:
Looks like the indentation got off here (and below) in this file.

Reply:
currently fixing